docs/kit/event-operations/battery-charging.md

```diff
@@ -43,7 +43,14 @@ Every volunteer performing the Battery Charging role must have read this documen
 
 The biggest risk of damage or injury is from the banana plugs coming unplugged from the charger. These are the red and black plugs that go into the side of the charger. All of the chargers should have the cables cable-tied to the charger to make it difficult to remove these plugs. Under no circumstances should these red and black plugs be unplugged from the chargers.
 
-The batteries themselves have many built-in safety features and, unless abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled by most fire extinguishers.
+The batteries themselves have many built-in safety features and, unless abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery:
+- DO NOT use a fire extinguisher - the batteries contain hazardous materials which cannot be safely tackled by most fire extinguishers.
+- DO NOT open or attempt to remove anything from the charging bag.
+ONLY IF SAFE TO DO SO:
```

Comment:
```suggestion

ONLY IF SAFE TO DO SO:
```
I assume the intention is for these to be separate lists

Reply:
It is a continuation of the list "In the extremely unlikely event of a fire due to a battery:", but with the extra qualifier of only if safe, so I'm not sure if the extra line is helpful or confusing?

Reply:
Without the new line the heading is appended to the last line of the previous list.